index.js

```diff
@@ -1,3 +1,6 @@
 const CVSS = require("./lib/cvss.js");
 
+const vector = CVSS("CVSS:3.0/AV:L/AC:H/PR:L/UI:R/S:C/C:L/I:L/A:L/E:U/RL:T/RC:R");
```

Comment:
I think you forgot to remove this here. Rest ist LGTM!

Reply:
Yes hahaha, I forgot to remove